beacon-chain/sync/initial-sync/service.go

```diff
@@ -58,9 +58,9 @@ type Service struct {
 	lastProcessedSlot uint64
 }
 
-// NewInitialSync configures the initial sync service responsible for bringing the node up to the
+// New configures the initial sync service responsible for bringing the node up to the
 // latest head of the blockchain.
-func NewInitialSync(ctx context.Context, cfg *Config) *Service {
+func New(ctx context.Context, cfg *Config) *Service {
```

Comment:
Can you change this back, this differs from all our other service. The current pattern is 
`NewServiceName`

Reply:
Will do!